.ci/ppc64le/lib_install_qemu_ppc64le.sh

```diff
@@ -71,5 +71,9 @@ build_and_install_qemu() {
         sudo -E make install
 
         sudo ln -sf $(command -v ${BUILT_QEMU}) "/usr/bin/qemu-system-${QEMU_ARCH}"
+
+        echo "Link virtiofsd to /usr/libexec/kata-qemu/virtiofsd"
+        sudo mkdir -p /usr/libexec/kata-qemu/
+        sudo ln -sf $(pwd)/virtiofsd /usr/libexec/kata-qemu/virtiofsd
```

Comment:
Does virtiofsd built by default? maybe worth verifying binary exist?

Reply:
@snir911 
Please review the updates for your comment. thanks!

Reply:
@jing-wang4 I think it will always be true, as the link still exist also if the virtiofsd doesn't, I'd test $(pwd)/virtiofsd before creating the link

Reply:
@snir911 
Updated, please review. thanks!

Reply:
@jing-wang4 LGTM, thanks!
Really nit, If you want to avoid output you can use "test -f" instead of "ls -l" find with me in any case, resolving